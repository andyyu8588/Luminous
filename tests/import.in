// For compiler/VM testing purpose

import tests/hard.in
import tests/methods.in

andyAction();
andyAction2();
print(job1.title);
