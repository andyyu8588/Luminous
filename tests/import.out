false
false
null
null
true
true
true
false
false
true
oktesttest123123
null
4.13
4.13
4.13
Andy did some workout.
Current weight: 
75.2
Andy aged.
Current age: 
19
Current height: 
182.5
Salary: 
300000
New job is: CEO
Salary: 
0
19
182.5
75.2
CEO
0
19
182.5
75.2
CEO
0
0
Andy did some workout.
Current weight: 
75
Andy aged.
Current age: 
20
Current height: 
184.5
Salary: 
0
New job is: CEO
Salary: 
0
20
184.5
75
CEO
0
20
184.5
75
CEO
0
bigman
